code/game/objects/structures/transit_tubes/transit_tube_construction.dm

```diff
@@ -0,0 +1,106 @@
+/obj/structure/construction/transit_tube
+	name = "uninstalled transit tube segment"
+	desc = "An uninstalled piece of a transit tube network."
+	anchored = FALSE
+	density = FALSE
+
+	icon = 'icons/obj/pipes/transit_tube_rpd.dmi'
+
+	var/installed_type = null
+	var/installed_type_flipped = null
+	var/flipped = FALSE
+
+/obj/structure/construction/transit_tube/Initialize(mapload)
+	. = ..()
+	update_icon_state()
```

Comment:
do we really need to update this on init?

Reply:
Nope, this can be killed if I set the icon_state for all the construction pieces